tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -182,6 +186,23 @@ describe('runs_table', () => {
     );
   }
 
+  function getColorGroupByHTMLElement(KEY: GroupByKey | REGEXEDIT) {
```

Comment:
KEY -> key. It does not have to be ALL_CAPS

Reply:
Please specify the return type, too.

Reply:
`REGEXEDIT` here is an object, not a literal `4`. IDK how it is matching against the switch case L199 but please change it to `key: GroupByKey | 'regexEdit'` or something like this.